.github/workflows/benchmark.yml

```diff
@@ -31,7 +22,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 18
```

Comment:
Set this to lts/* so we don't have to keep bumping it?

Reply:
I will anyway open an issue regarding general benchmark wf.